tests/acceptance/features/bootstrap/Provisioning.php

```diff
@@ -1441,8 +1441,11 @@ public function theAdministratorHasDeletedUserUsingTheProvisioningApi(?string $u
 	 */
 	public function theAdminDeletesUserUsingTheProvisioningApi(string $user):void {
 		$user = $this->getActualUsername($user);
-		$this->setResponse($this->deleteUser($user));
-		$this->pushToLastHttpStatusCodesArray();
+		$this->response = $this->deleteUser($user);
+		$this->pushToLastHttpStatusCodesArray(
+			(string) $this->response->getStatusCode()
+		);
```

Comment:
these should behave the same
```suggestion
		$this->setResponse($this->deleteUser($user));
		$this->pushToLastHttpStatusCodesArray();
```